src/core/types/Lab.ts

```diff
@@ -19,6 +19,8 @@ export interface GeneralLabView {
   pricePerShare: string;
   allowZapIn: boolean;
   allowZapOut: boolean;
+  zapInWith?: string;
+  zapOutWith?: string;
```

Comment:
Let's narrow these types

```suggestion
  zapInWith?: keyof Token['supported'];
  zapOutWith?: keyof Token['supported'];
```

Reply:
Added here https://github.com/yearn/yearn-finance-v3/pull/674/commits/7170e91d3685d21e399b3e874c82d785876763dd. Had a TODO after about this, but better change it now and refactor later with the import.